<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>ASCII</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>HEX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Packet Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="packetASCIIEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>751</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>ASCII representation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="packetHexEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>661</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>HEX representation</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>IPV4,IPV6,DNS Lookup</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="port">
    <property name="geometry">
     <rect>
      <x>312</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>1 to 65535</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>0.0/blank</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>130</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>TCP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>UDP</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SSL</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>90</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Load File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Resend Delay</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>911</width>
      <height>201</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Send</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Resend(sec)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To Port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Method</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hex</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search Saved Packets</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>170</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Saved Packet</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Persistent TCP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Log</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>456</x>
      <y>430</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Log Traffic</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>430</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Traffic Packet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>430</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Copy to Clipboard</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>911</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>From IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>From Port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To Port</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Method</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Error</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ASCII</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Hex</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
